lib/taskana-core/src/main/java/pro/taskana/common/internal/util/LogSanitizer.java

```diff
@@ -0,0 +1,18 @@
+package pro.taskana.common.internal.util;
+
+public class LogSanitizer {
+
+  private LogSanitizer() {
+    throw new IllegalStateException("Utility class");
+  }
+
+  /**
+   * Removes characters which break the log file pattern. Protects against injection attacks.
+   *
+   * @param loggingMessage String which should be sanitized
+   * @return sanitized logging message
+   */
+  public static String sanitizeLoggingMessage(Object loggingMessage) {
```

Comment:
class name is already LogSanitizer 
what about we rename this method to stripLineBreakingChars(Object loggingMessage)